### ISSUE TITLE
Необходимы возможности.

### ISSUE THREAD

**Issue by KostyaRKG** [GyverLibs/microDS3231#13]
Можно ли в вашей библиотеке устанавливать часовой пояс и изменять значение например + 1 день или - 1 день? Естественно день это как пример было бы круто делать тоже самое и с годом и месяцем ...

**Comment by GyverLibs** [comment]
нет, это библиотека для работы с чипом ds3231. Для манипуляций со временем есть библиотека Stamp
